.infrastructure/pv.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-data
+  namespace: todoapp
+spec:
+  storageClassName: standard
+  persistentVolumeReclaimPolicy: Delete
+  accessModes:
+    - ReadWriteMany
+  capacity:
+    storage: 1Gi
+  hostPath:
+    path: /data/
```

Comment:
Using 'hostPath' for PersistentVolumes is generally not recommended for production environments as it ties the volume to a specific node and can lead to data loss if the node fails. Consider using a cloud provider's persistent storage solution or a networked storage option for better reliability and data safety.